specs/protocol/rollup-node-p2p.md

```diff
@@ -348,6 +349,16 @@ A node may apply the block to their local engine ahead of L1 availability, if it
 - The subsequent forkchoice-update ensures this block is recognized as "unsafe"
   (see [fork choice updated](derivation.md#engine-api-usage))
 
+#### Branch selection
+
+Nodes expect that the sequencer will not equivocate, and therefore the fork choice rule for unsafe blocks
+is a "first block wins" model, where the unsafe chain will not change once it has been extended, unless
```

Comment:
```suggestion
is a FCFS model, where the unsafe chain will not change once it has been extended, unless
```
can this be expressed by standardised FCFS term instead? blocks pass through some kind of validation before being added right, so think "serve", in first come first _serve_, isn't the wrong word here, but open to that being debated

Reply:
I don't agree with "FCFS" being used here, because a later block won't ever be served if an earlier block was. Where FCFS describes an ordering, "First Block Wins" describes a selection.

I guess if you define "Serve" as "Consider this block" you could make the case, but if a competing block has already landed, no other block of the same height will be accepted, even if they are equally as valid as the original block.

Reply:
ok fair enough. was just an attempt to resort to standardised terminology wherever possible to lower barriers of entry as much as possible to devs familiar with the particular strategy but not with OP specs. nonetheless, for this particular situation, it does not seem to be possible based on your explanation, hence I rest my case.